### PR TITLE
Added in the begin and end route discontinuity attributes

### PR DESCRIPTION
**Pull request by kdiluca**

### CODE REVIEW

File: map-matching/api-reference.md [hunk 1]
```diff
@@ -247,6 +249,8 @@ Each `matched_point` may include:
 | `lon` | The longitude of the matched point. |
 | `type` | Describes the type of this match result - possible values include:<ul><li>`unmatched`</li><li>`interpolated`</li><li>`matched`</li></ul> |
 | `edge_index` | The index of the edge in the list of edges that this matched point is associated with. This value will not exist if this point was unmatched. |
+| `begin_route_discontinuity` | The boolean value is true if the begin location of the route is disconnected. This value will not exist if this is false. |
```
Comment (dgearhart) [inline map-matching/api-reference.md]: update wording as we discussed

**Comment by rmglennon** [comment]
:shipit: when ready!

**Comment by dgearhart** [comment]
@kdiluca looks good - we can merge today :shipit:

**Comment by kdiluca** [comment]
:shipit:  looks good

**Pull request merged**